testutil/iamauthtest/arn.go

```diff
@@ -0,0 +1,97 @@
+package iamauthtest
```

Comment:
I copied the things needed for a local AWS test server from Consul and put them here. It's verbatim code from Consul (aside from combining two files). This is what I've done for now. I think eventually we can extract the corresponding package in Consul, or place it in an external package that we can import.